src/gfnff/gfnff_eg.f90

```diff
@@ -612,7 +615,10 @@ subroutine gfnff_eg(env,pr,n,ichrg,at,xyz,makeq,g,etot,res_gff, &
 !       which is computed here to get the atomization energy De,n,at(n)
         call goed_gfnff(env,.true.,n,at,sqrab,srab,dfloat(ichrg),eeqtmp,cn,qtmp,eesinf,solvation,param,topo)
         de=-(etot - eesinf)
+      else
+         call timer%write(6)
```

Comment:
Is this for debugging?

Reply:
This one is added to print the calculation time for a single cycle during the gfnff geometry optimization.

![Gfnff_opt](https://user-images.githubusercontent.com/92109627/225016606-7b3d8ade-c0f0-48ea-9c6f-e87771688ba0.png)

Reply:
How about a *minpr* level compared to a full *pr* level of verbosity? (better variable names highly appreciated)

Reply:
What do you think about adding an optional argument to the `gfnff_eg` only for the case of optimization?
Otherwise, I do not see the point to change _pr_ into _minpr_ (or maybe _optpr_ as an alternative name).

Reply:
I have to agree with @awvwgk here. I don't see this as a necessary default printout and would prefer if this would only be written if ``verbose`` is specified. 
Additionally, it would be better if we make it more clear, that this is the single cycle time. ``total time`` might be misleading.

Reply:
So i added `minpr`/`optpr `option for geometry optimization + changed `total time` to `iter.time` for clarity.
Btw, I think we  should reassign global `printlevel` var to TSet class.

Reply:
@awvwgk, I added the changes you requested. 
@MtoLStoN, the output for gfnff optimization now looks like:

![image](https://github.com/grimme-lab/xtb/assets/92109627/77257068-719b-4ebd-a802-b2935297dac4)

PS: In this PR I added the commit 78fd1386db8dafc35edf001a8aba5461ddd41709 that only cleans and comments file `timer.f90`